nbs/07_quickstart_for_web_devs.ipynb

```diff
@@ -300,13 +301,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Advanced Static Media\n",
+    "## Other Static Media Files\n",
     "\n",
-    "If you want to customize your static media handler, you can define your own handler. \n",
+    "As you saw, `Script` and `Link` are specific to the most common static media use cases in web apps: including JavaScript and CSS.\n",
     "\n",
-    "```python\n",
+    "FastHTML also provides a way to serve static media files. You can define a route that uses `FileResponse` to serve the file at the specified path. This is useful for serving images, videos, and other media files.\n",
```

Comment:
By default, `fast_app` handles all media, including images, videos, and other media files. 

My original thought here is if the media is stored in an atypical spot, perhaps outside the FastHTML app in S3 or another system. Or within a seperate directory.

Reply:
Oh I see. That's cool! Let me revise this.